includes/options/class-amp-options-menu.php

```diff
@@ -109,6 +109,17 @@ public function add_menu_items() {
 			)
 		);
 
+		add_settings_field(
```

Comment:
This setting should only be added if `wp_using_ext_object_cache()` returns true, as we should only do response caching if we know that the cached value will persist across requests.

Reply:
True.  Good point.